### DISCUSSION TITLE
Node Graph plugin: Multiple links between two nodes

### DISCUSSION THREAD

**Discussion by maloy-ghosh** [grafana/grafana#58312] [Dashboards, Panels & Visualizations]
Is it possible to display multiple links between two nodes?

E.g. I am feeding this data over http endpoint (2 bidirectional link between two nodes)

```
{
  "request": {
    "method": "GET",
    "url": "api/datasources/proxy/3/nodegraphds/api/graph/data",
    "hideFromInspector": false
  },
  "response": {
    "edges": [
      {
        "id": "1",
        "mainStat": "53/s",
        "source": "1",
        "target": "2"
      },
      {
        "id": "2",
        "mainStat": "54/s",
        "source": "2",
        "target": "1"
      },
      {
        "id": "3",
        "mainStat": "55/s",
        "source": "1",
        "target": "2"
      },
      {
        "id": "4",
        "mainStat": "56/s",
        "source": "2",
        "target": "1"
      }
    ],
    "nodes": [
      {
        "arc__failed": 0.7,
        "arc__passed": 0.3,
        "detail__zone": "load",
        "id": "1",
        "subTitle": "instance:#2",
        "title": "Service1"
      },
      {
        "arc__failed": 0.5,
        "arc__passed": 0.5,
        "detail__zone": "transform",
        "id": "2",
        "subTitle": "instance:#3",
        "title": "Service2"
      }
    ]
  }
}
```

However the dashboard only displays one link (the later one)

![Capture](https://user-images.githubusercontent.com/6199322/200285800-0136337a-851c-4f07-bc80-fc54f2788ea0.PNG)

Is it possible to have both the links displayed?

Grafana version:                  grafana/grafana:8.0.2

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
